composer.json

```diff
@@ -20,9 +20,16 @@
         "php": ">=8.1"
     },
     "require-dev": {
-        "drupal/pathauto": "~1.0"
+        "drupal/pathauto": "~1.0",
```

Comment:
please remove these changes - see the message above

Reply:
Updated it.